cpp/tests/interop/arrow_utils.hpp

```diff
@@ -30,12 +30,80 @@
 #include <cudf/transform.hpp>
 #include <cudf/types.hpp>
 
+#include <arrow/api.h>
 #include <arrow/util/bitmap_builders.h>
 
-#include <algorithm>
-
 #pragma once
```

Comment:
Can you move this to the top of the file?

Reply:
Good eye, David.

Reply:
Done.

Reply:
I found one other instance of this in libcudf. Fixing here: https://github.com/rapidsai/cudf/pull/16636

Reply:
Thanks Bradley!